substrate/frame/identity/src/lib.rs

```diff
@@ -507,7 +507,7 @@ pub mod pallet {
 			match id.judgements.binary_search_by_key(&reg_index, |x| x.0) {
 				Ok(i) =>
 					if id.judgements[i].1.is_sticky() {
-						return Err(Error::<T>::StickyJudgement.into())
+						return Err(Error::<T>::StickyJudgement.into());
```

Comment:
How did these come back?
```suggestion
						return Err(Error::<T>::StickyJudgement.into())
```

Reply:
I think I know why, I changed my IDE rustfmt from nightly to stable as I was testing something. Changing it back.